gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -176,7 +185,18 @@ def densify(
             min_num_consistent_views=min_num_consistent_views,
         )
 
-        return dense_point_cloud, dense_point_colors
+        # Initialize densify metrics, add elapsed time per batch to the metric list
+        densify_metrics = GtsfmMetricsGroup(
+            name=METRICS_GROUP,
+            metrics=[
+                GtsfmMetric(name="num_batches/num_valid_views", data=len(loader)),
```

Comment:
the division operator makes this seem a bit like a fraction. How would you feel about
`num_valid_reference_views`? Technically we could batch up the inference differently (say 2 reference views at a time?), so I don't love the term batch here. Isn't there another batch dimension that we set to 1 in PatchmatchNet? Or am I remembering that wrong.

Reply:
Yes, it makes sense. I use the division operator because I am not very sure which is better. I will use `num_valid_reference_views`.

Reply:
resolved, thanks